### PR TITLE
Change SOFB variable names

### PR DESCRIPTION
**Pull request by fernandohds564**

### CODE REVIEW

File: siriuspy/tests/test_util.py [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
 import logging
-import numpy as np
 from unittest import mock, TestCase
+import numpy as np
+
 import siriuspy.util as util
 
 
-public_interface = (
+PUB_INTERFACE = (
```
Comment (xresende) [inline siriuspy/tests/test_util.py]: Should I convert `PUBLIC_INTERFACE` in all modules to simpler `PUB_INTERFACE`? Use of editors find-and-replace is much simpler if we keep standard as much as possible.

Comment (fernandohds564) [reply siriuspy/tests/test_util.py]: I think it's a good idea. but let me do it, I want to use `sed`! :joy:

Comment (fernandohds564) [reply siriuspy/tests/test_util.py]: done!

Comment (xresende) [reply siriuspy/tests/test_util.py]: nice. _grep_, _sed_, _code_... you choose your arm!

File: siriuspy/tests/ramp/test_waveform.py [hunk 2]
```diff
@@ -222,29 +218,27 @@ def test_update(self):
...
         """Test duration."""
         # TODO: implement it!
 
-
     def _test_update(self):
         """Test update."""
         # TODO: implement using mock to substitute server
         bo_b = WaveformDipole()
-        defoc = Waveform(maname='BO-Fam:MA-QD', dipole=bo_b)
+        defoc = Waveform(psname='BO-Fam:MA-QD', dipole=bo_b)
```
Comment (xresende) [inline siriuspy/tests/ramp/test_waveform.py]: @fernandohds564 , sure about this change?

Comment (fernandohds564) [reply siriuspy/tests/ramp/test_waveform.py]: @anacso17 ?

Comment (anacso17) [reply siriuspy/tests/ramp/test_waveform.py]: This is a right fix!

Comment (xresende) [reply siriuspy/tests/ramp/test_waveform.py]: @anacso17 , it was not that. I found strange that the name of the `Waveform` constructor argument is `psname` and the string passed is of _MA_ discipline. I guess this a side effect of abandoning MA IOCs. We will eventually finalize refactoring of classes that used _ maname_...

Comment (xresende) [reply siriuspy/tests/ramp/test_waveform.py]: @anacso17 , after you read my response, can you please resolve this conversation?

Comment (anacso17) [reply siriuspy/tests/ramp/test_waveform.py]: Oh, my mistake, the discipline is really wrong!

**Pull request merged**